docs/Consensus/ibft/ibft-parameters.md

```diff
@@ -0,0 +1,76 @@
+# IBFT parameters
```

Comment:
This should be moved out to the doc.goquorum project

Reply:
@bgravenorst, can you please check if we can reuse this page when looking at qbft documentation

Reply:
Yes, this can definitely be used.